src/endpoints/Stories/types.ts

```diff
@@ -180,6 +181,20 @@ export interface SlateStoryUpdateRequest extends GenericUpdateRequest {
     content_version?: number;
 }
 
+export interface PreviewResponse {
+    preview: {
+        content: {
+            'text/html': string;
+            'text/plain': string;
+        };
+    };
+}
+
+export interface PreviewOptions {
+    alignment?: Campaign.StoryAlignment;
+    appearance?: Campaign.StoryAppearance;
```

Comment:
So no re-exporting? :thinking:

Reply:
Oh, ok. Didn't think it's necessary, but if it would be helpful, re-exported. 👍🏻